### ISSUE TITLE
AudioSet temporally-strong labels

### ISSUE THREAD

**Issue by knma1992** [fschmid56/EfficientAT#30]
Firstly I would like to state that this repo is great, so many models, all in pytorch and getting them to work on my machine was very easy.

Have you tried fine-tuning the models on the temporally-strong labeled subset of the AudioSet dataset?

**Comment by fschmid56** [comment]
To follow up on this: we have fine-tuned a couple of transformer models on the temporally-strong labels of AudioSet:

https://github.com/fschmid56/PretrainedSED

The models are not yet low-complexity, but we will also publish something in that regard soonish :-)

**Comment by zuowanbushiwo** [comment]
> The models are not yet low-complexity, but we will also publish something in that regard soonish :-)

Looking forward to your less complex models :-)
